lib/ryan.rb

```diff
@@ -18,9 +18,25 @@ class Ryan
   def_delegators :const,
     :name, :funcs, :type, :initialization_args, :func_by_name, :class?, :module?
 
+  def self.root
```

Comment:
Good suggestion! I like the idea of having a `root` provided by gems, especially if they deal a lot with files!